packages/cardano-services/src/Http/HttpServer.ts

```diff
@@ -41,7 +44,10 @@ export abstract class HttpServer extends RunnableModule {
     this.app = express();
     // must use this before router
     this.app.use(
-      bodyParser.json({ reviver: (key, value) => (key === '' ? util.fromSerializableObject(value) : value) })
+      bodyParser.json({
+        limit: this.config.bodyParser?.limit || '500kB',
+        reviver: (key, value) => (key === '' ? util.fromSerializableObject(value) : value)
```

Comment:
This means that only the last attribute of the rquest will be parsed by `fromSerializableObject`?

Reply:
I don't think so, tested this:
```js
JSON.parse('{"1": 1, "2": 2, "3": {"4": 4, "5": {"6": 6}}}', (key, value) => {
  if (key === '') console.log(`"${key}"`, value); // log the current property name, the last is "".
  return value;     // return the unchanged property value.
});
```
prints `"" {1: 1, 2: 2, 3: {…}}`

It's the last call, but as I understand it's for mapping the entire object.